exporter/awsemfexporter/metric_translator.go

```diff
@@ -32,8 +34,46 @@ const (
 	containerInsightsReceiver = "awscontainerinsight"
 	attributeReceiver         = "receiver"
 	fieldPrometheusMetricType = "prom_metric_type"
+
+	// Entity fields
+	keyAttributeEntityServiceName           = "aws.entity.service.name"
+	serviceName                             = "Name"
+	keyAttributeEntityDeploymentEnvironment = "aws.entity.deployment.environment"
+	deploymentEnvironment                   = "Environment"
+	keyAttributeEntityType                  = "aws.entity.type"
+	entityType                              = "Type"
+	service                                 = "Service"
+	resource                                = "Resource"
+	keyAttributeEntityResourceType          = "aws.entity.resource.type"
+	resourceType                            = "ResourceType"
+	keyAttributeEntityIdentifier            = "aws.entity.resource.identifier"
+	identifier                              = "Identifier"
+	attributeEntityCluster                  = "aws.entity.k8s.cluster.name"
+	cluster                                 = "Cluster"
+	attributeEntityNamespace                = "aws.entity.k8s.namespace.name"
+	namespace                               = "Namespace"
+	attributeEntityWorkload                 = "aws.entity.k8s.workload.name"
+	workload                                = "Workload"
+	attributeEntityNode                     = "aws.entity.k8s.node.name"
+	node                                    = "Node"
 )
 
+var keyAttributeEntityToShortNameMap = map[string]string{
+	keyAttributeEntityType:                  entityType,
+	keyAttributeEntityResourceType:          resourceType,
+	keyAttributeEntityIdentifier:            identifier,
+	keyAttributeEntityServiceName:           serviceName,
+	keyAttributeEntityDeploymentEnvironment: deploymentEnvironment,
+}
+
+var attributeEntityToShortNameMap = map[string]string{
+	attributeEntityCluster:   cluster,
+	attributeEntityNamespace: namespace,
+	attributeEntityWorkload:  workload,
+	attributeEntityNode:      node,
+	// TODO: add attributes for EC2
```

Comment:
should this TODO be here? When is it getting addressed?

Reply:
Yes, currently we are not exporting ec2 attributes under the fields `aws.entity.<...>`. I believe we have a task to track it, if not I will cut one.

Reply:
okay lets just make sure this gets done